### PR TITLE
Event/Brewfest: implement relay race

### PR DESCRIPTION
**Pull request by SnapperRy**
**Changes proposed**:

More info [here](http://wowwiki.wikia.com/wiki/Brewfest#Continuing_to_help_out_Brewfest).

This is the race that you can try every 18 hours to get more tokens (2 for each barrel delivered).

**Target branch(es)**: 335

**Tests performed**: tested and working.

### CODE REVIEW

**Comment by tkrokli** [comment]
Looks like you have added the `Edit` action in your wowwiki link. I hope that was unintentional. :-p
- http://wowwiki.wikia.com/wiki/Brewfest#Continuing_to_help_out_Brewfest
- http://wow.gamepedia.com/Brewfest#Continuing_to_help_out_Brewfest

**Comment by SnapperRy** [comment]
@tkrokli: it was intentional actually :P wasn't sure how to link directly to that paragraph, and the edit one seemed a good choice rather than "go there and search for this string".

Changed the link to yours, thanks :)

File: src/server/scripts/Spells/spell_holiday.cpp [hunk 1]
```diff
@@ -1011,6 +1013,40 @@ class spell_brewfest_relay_race_intro_force_player_to_throw : public SpellScript
...
+    spell_brewfest_relay_race_turn_in() : SpellScriptLoader("spell_brewfest_relay_race_turn_in") { }
+
+    class spell_brewfest_relay_race_turn_in_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_brewfest_relay_race_turn_in_SpellScript);
+
+        void HandleDummy(SpellEffIndex effIndex)
+        {
+            PreventHitDefaultEffect(effIndex);
+            if (GetHitUnit()->HasAura(SPELL_SWIFT_WORK_RAM))
```
Comment (GitVerge) [inline src/server/scripts/Spells/spell_holiday.cpp]: I think that you don't need this if

File: sql/updates/world/3.3.5/9999_99_99_99_world.sql [hunk 2]
```diff
@@ -0,0 +1,63 @@
+--
+DELETE FROM `spell_script_names` WHERE `spell_id`=43755;
```
Comment (GitVerge) [inline sql/updates/world/3.3.5/9999_99_99_99_world.sql]: delete by scriptname, not spell id

**Comment by Kittnz** [comment]
@SnapperRy can we have the changes so we can merge? Thanks!

**Comment by Aokromes** [comment]
atm he lacks internet, retard italian isp xd

**Comment by SnapperRy** [comment]
Sorry for the delay, still without a stable connection at home. Update, should be fine to merge.

**Pull request merged**